### DISCUSSION TITLE
user-guide/ms-word-plugin

### DISCUSSION THREAD

**Discussion by giscus** [zotero-chinese/wiki#166] [文档评论区]
# user-guide/ms-word-plugin

这些是使用 Zotero for MS Office Word 插件的说明。 要在 WPS 中使用插件，请参见 。 Zotero Word 插件选项卡 Zotero Word 插件选项卡Zotero Word 插件选项卡 一般情况下，Zotero 安装时会安装 Zotero Word 插件，其会在 Microsoft Word 里添加一个 Zotero...

https://zotero-chinese.com/user-guide/ms-word-plugin.html

**Comment by Namelesswall** [comment]
请问在word中增加参考文献，在设置首选项时选中的样式没有“引注显示为脚注/尾注”的选项，只有“引注存储为”是为什么呢？

**Reply by zepinglee** [reply]
需要选择注释类（note）CSL 样式，非注释类没有这个选项。

**Comment by ailingwang999** [comment]
想要在文章插图的题注中添加参考文献，为什么不可以呀，显示“此处无法插入zotero字段”

**Reply by northword** [reply]
题注需要是正文里的，不能是文本框里的（通常浮动式图片插入题注时，题注会在文本框。

**Reply by cypans** [reply]
有什么办法能在文本框里插入引用吗？因为表格是在文本框里的，然后表格中的一些模型需要引用

**Comment by Sherry520** [comment]
不可以直接插入选中的条目吗？不想用那个搜索

**Reply by northword** [reply]
z7上，选中一个条目，在word里点击插入引注后，这个条目会显示在候选区，直接回车即可。

**Reply by listenCC** [reply]
针对“northword”的回答补充一下，插入引注的形式应该是快速引注，经典视图不可以。

**Comment by Sherry520** [comment]
好，那我升级7
1271803144
***@***.***
---- Replied Message ----
***@***.***>Date5/10/2024 ***@***.***>CcYuhang ***@***.***>,
***@***.***>SubjectRe: [zotero-chinese/wiki] user-guide/ms-word-plugin (Discussion #166)
z7上，选中一个条目，在word里点击插入引注后，这个条目会显示在候选区，直接回车即可。

**Comment by iampearl918** [comment]
请问能否、以及如何能将所有word文档中插入的引文一起调出来，进行编辑？而且还能定位到word文档中相应位置（类似endnote）

此外，目前只给出了由“文中”到“参考文献列表”的单链添加方式，能否添加反链？我感觉反链也很重要，可以快速定位到word文中引用的位置，如果最后提交前要修改也便于查找。

**Reply by northword** [reply]
citation插件可以实现在 zotero 中查看某文档中的所有引注，并显示其引注序号。

目前无法实现从参考文献表链接到引注。

也无法实现类似 endnote 那种在 word 中弹窗集中管理所有引注。

**Reply by ZQ-starry** [reply]
1、Zotero 7可以实现引注和文献目录之间双向超链接了？
2、或者Zotero 7可以实现同时引用多个引注时不仅仅只是链接最后一个，而是准确链接所选引注？

**Comment by GGGGGGGGGGN** [comment]
请问可以添加超链接吗，目前ctrl+左键点不过去

**Reply by northword** [reply]
https://zotero-chinese.com/user-guide/misc/link-citation-to-bibliography

**Reply by GGGGGGGGGGN** [reply]
太酷了

**Comment by sweetmomojam** [comment]
想要直接生成参考文献列表，但是出现“执行本操作前您需要插入引注”。。。

**Comment by prong879** [comment]
请问zotero7是否支持以页下注著录格式展示参考文献列表？如果支持，请问如何操作？

**Comment by 7yzx** [comment]
在word中插入参考文献的上角标是很小的数字1，而不是[1]，这该如何处理啊

**Reply by northword** [reply]
这是期刊样式要求的，你可以换一个引文风格试一下。

**Comment by ZhaoHanQing1999** [comment]
请问如果是在写毕业论文的时候，综述和正文部分都需要参考文献，是不是只能通过写完一部分再转成纯文本的方式实现两个部分的参考文献都从1开始标注

**Reply by northword** [reply]
是的，或分两个文档。

**Reply by sbaiusai** [reply]
感觉这个功能很需要啊，分节引用参考文献，找了很久都没有相应的插件，官方论坛也有很多人说这个问题

**Comment by masterofyi** [comment]
请问我在一处插入多篇文献时，会在一个序号内包括多篇文献的信息。
比如，我插入3篇文献，他在正文显示［ 1］，而不是［ 1-3］，然后脚注显示为［ 1］：作者1，文献名称1，出版社1，日期1；作者2，文献名称2，出版社2，日期2；作者3，文献名称3，出版社3，日期3。
我想让他显示为：［ 1］：作者1，文献名称1，出版社1，日期1；［ 2］：作者2，文献名称2，出版社2，日期2；［ 3］：作者3，文献名称3，出版社3，日期3

**Comment by Jiaruident** [comment]
请教一下，前两天添加参考文献用的好好的，昨天开始点击Add/Edit Citation没有反应了是怎么回事呀[大哭]

**Reply by northword** [reply]
禁用所有插件后重试呢？

**Reply by Jiaruident** [reply]
禁用后可以了，感谢！一旦打开插件又不行了...

**Reply by northword** [reply]
可能是citation插件？并尝试更新所有插件和 zotero 版本。

**Reply by 935485852** [reply]
请问 在我的库中打开 这个按钮点了没反应是为什么呀

**Comment by 935485852** [comment]
请问 在我的库中打开 这个按钮点了没反应是为什么呀

**Comment by zps2022** [comment]
请问如何在word中表格添加参考文献呀？为什么我添加之后不显示呢？而且似乎只能以表格为一个整体添加引用，没法在表格不同地方添加不同的引用

**Comment by xiaolv-dd** [comment]
我的在显示参考文献的时候就会跳出“替换这个这个zotero字段吗？”然后确定以后显示出来的格式就不一样 这是怎么回事呀？他是第一个参考文献不换行，其他的会换行

**Comment by PitKing01** [comment]
能不能在每一章最后生成本章的参考文献表，好像endnote就有这个选项

**Reply by northword** [reply]
Zotero 暂时无法实现，如果一定要，可以一章写一个文件。

**Comment by Kkkkk1666** [comment]
Style Error: GB/T 7714—2015（顺序编码，双语，无 URL、DOI）
在word文档中的脚注中，我设置了每一页重新编号。但是跨页中对同一文献的引用中，第二处则会显示，“同1”或“同上”，这样会引起人的误解，因为它上方的文献并不是同一个。这种问题应该如何处理？

**Comment by 2026-lx** [comment]
为什么插入引注显示为[1,2],而且参考文献列表也没有对齐。我想要的是[1-2]最后只有序号在前，其他文本都是对齐的那种，我用的是7714-2015（顺序编号……无DOI、这个）

**Comment by trr19991226-lab** [comment]
在写论文返修说明的时候，需要在每个问题的回应上单独插入参考文献列表而不是最后统一插入参考文献，这种怎么操作呢

**Comment by YozhenMa** [comment]
请问为什么我在word中生成bibliography，例如：“Wiese, F., & Baldini, M. (2018). Conceptual model of the industry sector in an energy system model: A case study for Denmark. Journal of Cleaner Production, 203, 427–443. (1 📊). https://doi.org/10.1016/j.jclepro.2018.08.229”，中间会有(1 📊)，我返回zotero看文献信息，是存档位置，请问这个该怎么解决，正常文献的bibliography不应该会有这个内容。

**Reply by northword** [reply]
你使用了zotero one 之类的或是其他插件，它不合规范的占用「存档」等字段来存储影响因子或是引用量之类的。需要清除这些字段里的内容。

**Reply by YozhenMa** [reply]
您好，请问我该如何排查是哪个插件导致的问题，我找了很久没有找到这个问题的所在，谢谢！

**Reply by northword** [reply]
> 您好，请问我该如何排查是哪个插件导致的问题，我找了很久没有找到这个问题的所在，谢谢！
> 

大概率是zotero one或zotero if，市面上还在积极维护的插件就这俩在不合理占用字段。或者你发一下你装了哪些插件。
但无论怎样，你都只能手动清理这些字段。或者写个脚本清理。

**Reply by YozhenMa** [reply]
你好！我看了下我的插件，里面有：蒲公英、Actions and Tags for Zotero、Awesome GPT、Better Notes for Zotero、Chartero、Easier Citation、Ethereal Reference、Ethereal Style、Green Frog、Jasminum、Linter for Zotero、 Zotcard、zotero插件市场。

**Reply by northword** [reply]
这些插件应该都不会写这个字段，那可能就是以前搞的了？不影响，那就手动清理一下这个字段吧，linter 倒是在考虑做这个功能，但还没搞。

**Reply by northword** [reply]
或者就是直接编辑 CSL 文件，把 archive （应该是？我也不确定这个在 CSL 里的字段名叫啥）相关的给删掉

**Reply by YozhenMa** [reply]
好的好的 十分感谢 我去研究一下！！谢谢！

**Reply by northword** [reply]
linter 现已经支持清理这些内容，可以更新到 3.1.0 ，然后对需要清理的条目右击 - linter - lint and fix。
